README.md

```diff
@@ -8,6 +8,30 @@ The site is built automatically with every push thanks to a [GH Actions](https:/
 
 If you want to build it locally, follow these simple steps:
 
+If you use windows, please execute
+
+```js 
+npm install -g gifsicle
+```
+perhaps
+
+```js
+cnpm install gifsicle
+```
+
+If you use Linux, please execute
+
+```shell
+apt-get update
+apt-get install dh-autoreconf
+```
+
+If you use MacOS, please execute it first
+
+```shell
+brew install automake
+```
+
 ```js
```

Comment:
This should then be seperated as it applies to all OS (as it seems).
So after the section `dependencies`, a section `building the site` should follow.